src/test/java/core/basesyntax/service/impl/WriteServiceImplTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.service.WriteService;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class WriteServiceImplTest {
+    private static final String PATH_TO_FILE = "src/resources/testReport.csv";
+
+    private static WriteService writeService;
+
+    @BeforeAll
+    static void setUp() {
+        writeService = new WriteServiceImpl();
+    }
+
+    @Test
+    void writeToFile_writePathToFile_ok() {
+        String report = "fruit,quantity" + System.lineSeparator()
+                + "banana,10" + System.lineSeparator()
+                + "apple,40";
+        writeService.writeToFile(PATH_TO_FILE, report);
+        String actualReport = readFromFile();
```

Comment:
I guess if you test "writePathToFile" method you should not use another methods from your code (You shouldn't use readFromFile())

Reply:
It`s for the KISS principle. I created a new method here outside of my other classes. Maybe, you are right, but I'll leave it like this for now.